src/data/perf_stat.rs

```diff
@@ -202,13 +202,13 @@ impl CollectData for PerfStatRaw {
                             if let Some(os_error) = e.downcast_ref::<std::io::Error>() {
                                 match os_error.kind() {
                                     ErrorKind::NotFound => {
-                                        error!("Instance does not expose Perf counters")
+                                        warn!("Instance does not expose Perf counters")
                                     }
                                     _ => match os_error.raw_os_error().unwrap() {
-                                        libc::EMFILE => error!(
-                                            "Too many open files. Increase limit with `ulimit -n`"
+                                        libc::EMFILE => warn!(
```

Comment:
Wouldn't this be needed for any basic collection to work in APerf? So, this has to be an error and not warning?

Reply:
It doesn't prevent APerf from running.